clients/docs/java-springboot.rst

```diff
@@ -0,0 +1,176 @@
+
+   .. code-block:: text
+
+         ...
+         2020-02-13 14:41:58.248  INFO 44191 --- [ntainer#0-0-C-1] i.c.e.c.c.springboot.ConsumerExample     : received alice {"count": 0}
+         2020-02-13 14:41:58.248  INFO 44191 --- [ntainer#0-0-C-1] i.c.e.c.c.springboot.ConsumerExample     : received alice {"count": 1}
+         2020-02-13 14:41:58.248  INFO 44191 --- [ntainer#0-0-C-1] i.c.e.c.c.springboot.ConsumerExample     : received alice {"count": 2}
+         2020-02-13 14:41:58.248  INFO 44191 --- [ntainer#0-0-C-1] i.c.e.c.c.springboot.ConsumerExample     : received alice {"count": 3}
+         2020-02-13 14:41:58.249  INFO 44191 --- [ntainer#0-0-C-1] i.c.e.c.c.springboot.ConsumerExample     : received alice {"count": 4}
+         2020-02-13 14:41:58.249  INFO 44191 --- [ntainer#0-0-C-1] i.c.e.c.c.springboot.ConsumerExample     : received alice {"count": 5}
+         2020-02-13 14:41:58.249  INFO 44191 --- [ntainer#0-0-C-1] i.c.e.c.c.springboot.ConsumerExample     : received alice {"count": 6}
+         2020-02-13 14:41:58.249  INFO 44191 --- [ntainer#0-0-C-1] i.c.e.c.c.springboot.ConsumerExample     : received alice {"count": 7}
+         2020-02-13 14:41:58.249  INFO 44191 --- [ntainer#0-0-C-1] i.c.e.c.c.springboot.ConsumerExample     : received alice {"count": 8}
+         2020-02-13 14:41:58.249  INFO 44191 --- [ntainer#0-0-C-1] i.c.e.c.c.springboot.ConsumerExample     : received alice {"count": 9}
+
+#. When you are done, press ``kbd:[CTRL-C]``.
+
+#. View the :devx-examples:`producer|clients/cloud/java-springboot/src/main/java-springboot/io/confluent/examples/clients/cloud/springboot/kafka/ProducerExample.java` and :devx-examples:`consumer code|clients/cloud/java-springboot/src/main/java-springboot/io/confluent/examples/clients/cloud/springboot/kafka/ConsumerExample.java`.
+
+Kafka Streams
+-------------
+
+The |kstreams| API reads the same topic and does a stateful sum aggregation,
+also a rolling sum of the counts as it processes each record.
+
+#. Run the |kstreams| application:
+
+   .. code-block:: text
+
+      ./startStreams.sh
+
+   You should see:
+
+   .. code-block:: text
+
+         ...
+         [Consumed record]: alice, 0
+         [Consumed record]: alice, 1
+         [Consumed record]: alice, 2
+         [Consumed record]: alice, 3
+         [Consumed record]: alice, 4
+         [Consumed record]: alice, 5
+         [Consumed record]: alice, 6
+         [Consumed record]: alice, 7
+         [Consumed record]: alice, 8
+         [Consumed record]: alice, 9
+         ...
+         [Running count]: alice, 0
+         [Running count]: alice, 1
+         [Running count]: alice, 3
+         [Running count]: alice, 6
+         [Running count]: alice, 10
+         [Running count]: alice, 15
+         [Running count]: alice, 21
+         [Running count]: alice, 28
+         [Running count]: alice, 36
+         [Running count]: alice, 45
+         ...
+
+#. When you are done, press ``kbd:[CTRL-C]``.
```

Comment:
Should this read just `<ctrl-c>`?

Reply:
Correct! Good catch. Syntax error with the migration